week-1/K-functions-parameters/exercise3.js

```diff
@@ -1,4 +1,7 @@
 // Write your function here
+function createGreeting(name) {
+    return "my name is  " + name;
```

Comment:
Not exactly a "greeting" as it doesn't "greet" anyone, just states a name ;)

Reply:
changed it.thanks